### ISSUE TITLE
Particle Emitters - "Change the capacity of..." event NOT working

### ISSUE THREAD

**Issue by erdo-dp** [4ian/GDevelop#2821]
## Describe the bug
Particle Emitters capacity value are not being changed updated trough the "Change the capacity of..." event. The original value persist.

## Example to Reproduce
[game.json.zip](https://github.com/4ian/GDevelop/files/6836117/game.json.zip)

## Other details
* All betas since GDevelop 5 exist

**Comment by Bouh** [comment]
Seem this isn't working, I can't found `totalParticleCount` in emitter with the debugger on a breakpoint.

```JS
 getTotalParticleCount(): integer {
      return this.emitter.totalParticleCount;
    }
```

[The documentation of PIXI.Particles ](http://pixijs.io/pixi-particles/docs/classes/emitter.html )confirm there is no total `totalParticleCount` in emitter.

**Comment by 4ian** [comment]
Might be something dating from the port from SFML SPARK particles to pixi particles.

**Comment by AlexandreSi** [comment]
Thanks for reporting this bug, the particle emitter needed a good clean!

**Comment by tristanbob** [comment]
So.... is changing the capacity of an emitter supposed to work now?  

https://user-images.githubusercontent.com/8879811/139517935-dbf271d6-c4cf-4826-8b5a-b34bf44dd148.mp4

(This is using the web version of GDevelop)

**Comment by 4ian** [comment]
It should work, the emitter should stop emitting after `capacity/flow` seconds.
I've just tested on the web app:
- setting the capacity to 45 and flow to 45 will make the emitter last 1 second (ok!)
- change the capacity to 90 in the events will make the emitter last 2 second (ok!)
- change the capacity to 10 in the events will make the emitter last <1 second (ok!)
- setting the capacity to -1 and changing it to 10 in the events will make the emitter last <1 second (ok!)

In other words: it seems to work well for me?

**Comment by tristanbob** [comment]
I should have tested this more.  On my desktop (5.0.121) this works great. 

When I tested (again) on the web version, it had updated to that version and of course, it worked. My first test must have been on an unfixed version.

Thanks @AlexandreSi for your work on this!

**Comment by 4ian** [comment]
No problem, this is actually a good sign we need a notification in the web-app that an update is available! :)
